english/smash-spoilers/smash-spoilers.md

```diff
@@ -0,0 +1,3 @@
+... then you walk trhoug looking for a real option that make you finish
+your work... trying to avoid any suspicion. Then you see a new door. One that recognize and you know it can be fun... you don't know WHAT IS AT THE END... so you enter the breaking bad door ...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.